build.sbt

```diff
@@ -110,7 +110,9 @@ val core = MultiScalaCrossProject(
     buildInfoSettings,
     libraryDependencies ++= List(
       scalameta.value,
-      "org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided
+      "org.scala-lang" % "scala-reflect" % scalaVersion.value % Provided,
+      "org.scala-lang" % "scala-compiler" % scalaVersion.value,
```

Comment:
Do we use scala-compiler?

Reply:
I forgot to remove it. I wanted to keep a trace of the experimentation in the git history: https://github.com/scalacenter/scalafix/pull/594/commits/3e11aeac950f2246e7a52a04c319218ccbbe2c97